tools/cellxgene_census_builder/src/cellxgene_census_builder/release_manifest.py

```diff
@@ -117,9 +118,27 @@ def _validate_release_info(
     if rls_info["release_build"] != rls_tag:
         raise ValueError("release_build must be the same as the release tag")
 
-    if rls_info["soma"] != {"uri": urlcat(census_base_url, rls_tag, "soma/"), "s3_region": CENSUS_AWS_REGION}:
+    from urllib.parse import urlparse
+
+    parsed_url = urlparse(census_base_url)
+    prefix = parsed_url.path
+
+    expected_soma_locator = {
+        "uri": urlcat(census_base_url, rls_tag, "soma/"),
+        "relative_uri": urlcat(prefix, rls_tag, "soma/"),
+        "s3_region": CENSUS_AWS_REGION,
+    }
+    expected_h5ads_locator = {
+        "uri": urlcat(census_base_url, rls_tag, "h5ads/"),
+        "relative_uri": urlcat(prefix, rls_tag, "h5ads/"),
+        "s3_region": CENSUS_AWS_REGION,
```

Comment:
`s3_region` is effectively a deprecated field, so I think hardcoding it makes sense. It will always point to the default bucket, which I don't think will ever be moved from `us-west-2`.

Reply:
OK, not a big deal.  But given that the _bucket_ is defined in the config, I'm not sure why the region isn't also defined in the config.  They are tightly linked configuration values.....